tests/datasets/test_custom_dataset.py

```diff
@@ -29,13 +28,15 @@ def test_custom_dataset(step_lr, optimizer, tokenizer, get_model, train, mocker)
     args, kwargs = train.call_args
     train_dataloader = args[1]
     eval_dataloader = args[2]
+    tokenizer = args[3]
     
-    VAL_SAMPLES = 818
-    TRAIN_SAMPLES = 14732
-    CONCAT_SIZE = 2048
+    assert len(train_dataloader) == 226
+    assert len(eval_dataloader) == 2*226
+
+    STRING = tokenizer.decode(next(iter(train_dataloader))["input_ids"][0], skip_special_tokens=True)
+    EXPECTED_STRING = "[INST] Напиши функцию на языке swift, которая сортирует массив целых чисел, а затем выводит его на экран [/INST] Вот функция, "
```

Comment:
Will be better to use English example

Reply:
Sure, this was just the first example in the resulting dataset. Here we just want to see that the instruction tags are added correctly. But can select a different example as well.